### ISSUE TITLE
Problem with multisite-lists form entry

### ISSUE THREAD

**Issue by DavidRoy** [BiologicalRecordsCentre/iRecord#619]
From this iRecord forum post.  
https://www.brc.ac.uk/irecord/comment/reply/7759

It may relate to recent dynamic mapping work.  The multisite-lists page is not the most widely used page but a few people use it as their primary approach to data entry.
https://www.brc.ac.uk/irecord/enter-multisite-lists

**Comment by burkmarr** [comment]
From iRecord post: 

> Is there a known issue with editing multi-site records? I've tried a number of times to edit a record which was submitted as part of a multi-site list of records, but get a map with no records on it, It asks me to click the records I want to modify or move etc, however nothing appears despite zooming out and in (just incase it was to do with the scale). I've been using Chrome as my browser.

**Comment by johnvanbreda** [comment]
This does seem to be a dynamic map issue, as if I change my layer to OSM then reload the page, the grid squares I entered appear correctly. @burkmarr I'll leave this in your safe hands but come back to me if you want a hand.

**Comment by burkmarr** [comment]
@johnvanbreda - what's the workflow here? (I am not familiar with this form.) Presumably you can only edit records for the current batch - i.e. those that are 'finished' but not submitted? Or have I misunderstood? I don't have any problems seeing/editing 'finished' but unsubmitted records on dev, regardless of what map I am displaying. When I go back to the same place after submission, I don't see previously submitted records, but that's what I would expect.

**Comment by johnvanbreda** [comment]
The problem occurs only after saving the whole form (i.e. finishing one or more grid square batches, then saving the form), then finding one of the records in My Records and clicking Edit. If you masquerade as me you can look at My Records and the last few records were entered using the multisite form.

**Comment by burkmarr** [comment]
Okay thanks @johnvanbreda - I got it now. Will work on this early next week at the latest.

**Comment by burkmarr** [comment]
Posted on the forum thread so user has the workaround in the meantime.

**Comment by burkmarr** [comment]
Bug found and fixed. Pull request made on devel.

**Comment by burkmarr** [comment]
Merged into devel.

**Comment by burkmarr** [comment]
Merged to master. https://github.com/Indicia-Team/media/commit/4c46fb9ff4060074db1b25059b023e1c9f908830
